packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -0,0 +1,38 @@
+/* Common Tag styles */
+.tag {
+  display: flex;
+  flex: none;
+  flex-direction: row;
+  gap: var(--size-tiny);
+  align-items: center;
+  justify-content: center;
+
+  padding: var(--size-tiny) var(--size-xs);
+  border-radius: var(--size-m);
+  color: var(--text-color);
+  background-color: var(--bg-color);
+
+  --text-color: var(--color-text--default);
+  --bg-color: var(--color-surface--3);
+}
+
+.tag:hover {
+  --bg-color: var(--color-surface--4);
+}
+
+/* Styles for Tag's text */
+.tagLabel {
+  font-weight: 700;
+  font-size: var(--text-xxs);
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  line-height: var(--text-xs);
```

Comment:
Using `rem` as a unit for `line-height` is problematic, since it's value gets computed from the root font-size, not from where it's declared in.

It's preferred to use unitless values if you need to set it to something specific, or just let it inherit the default value, `normal`, which translates to `1.2`. The way unitless values work with `line-height` is that they multiply the element's font-size with itself.

Reply:
Refactored to match the design and use em